### PR TITLE
Added tests for dvr router type change

### PR DESCRIPTION
**Pull request by TregubovYury**
Author: Tregubov Yury

Added tests for QA-311 QA-312 QA-313 QA-315

Also updated TestRail ids for my previous test in the ovs restart

### CODE REVIEW

File: mos_tests/neutron/python_tests/test_dvr.py [hunk 1]
```diff
@@ -64,3 +65,180 @@ def test_vm_connectivity_wo_floating(self, variables):
             security_groups=[self.security_group.id])
 
         self.check_ping_from_vm(server, vm_keypair=self.instance_keypair)
+
+    def try_to_update_router_to_centralized(self, router_id):
```
Comment (gdyuldin) [inline mos_tests/neutron/python_tests/test_dvr.py:261]: Do you expect exception? If yes - let's use this check - https://pytest.org/latest/assert.html#assertions-about-expected-exceptions

File: mos_tests/neutron/python_tests/test_dvr.py [hunk 2]
```diff
@@ -64,3 +65,180 @@ def test_vm_connectivity_wo_floating(self, variables):
...
+                    id=router_id)['routers'][0]
+        err_msg = 'distributed parameter for the router {0} is {1}'.format(
+                   router['name'], router['distributed'])
+        err_msg = ''.join((err_msg, 'But it expeced to be True'))
+        assert router['distributed'], err_msg
+
+        self.try_to_update_router_to_centralized(router['id'])
+
+    @pytest.fixture
+    def config_legacy_router(self, init):
```
Comment (gdyuldin) [inline mos_tests/neutron/python_tests/test_dvr.py]: Only router?

File: mos_tests/neutron/python_tests/test_dvr.py [hunk 3]
```diff
@@ -17,6 +17,7 @@
 import pytest
 
 from mos_tests.neutron.python_tests import base
+from neutronclient.common.exceptions import NeutronClientException
```
Comment (rpromyshlennikov) [inline mos_tests/neutron/python_tests/test_dvr.py:21]: should be:
 import pytest

 from neutronclient.common.exceptions import NeutronClientException

 from mos_tests.neutron.python_tests import base

File: mos_tests/neutron/python_tests/test_dvr.py (hunk 2 above)
Comment (TregubovYury) [inline mos_tests/neutron/python_tests/test_dvr.py]: Yes, only router is created in this test, according to the original scenario.

File: mos_tests/neutron/python_tests/test_dvr.py (hunk 1 above)
Comment (TregubovYury) [inline mos_tests/neutron/python_tests/test_dvr.py:261]: mmm totally forgot that such frameworks usually have the expected exception feature

File: mos_tests/neutron/python_tests/test_dvr.py (hunk 2 above)
Comment (gdyuldin) [inline mos_tests/neutron/python_tests/test_dvr.py]: But in method called `config_legacy_router` creates networks, instances, etc...

Comment (TregubovYury) [inline mos_tests/neutron/python_tests/test_dvr.py]: this test is for QA-311 QA-312
config_legacy_router is for QA-313 QA-315

File: mos_tests/neutron/python_tests/test_dvr.py (hunk 1 above)
Comment (TimurNurlygayanov) [inline mos_tests/neutron/python_tests/test_dvr.py:261]: ok let's use them.

File: mos_tests/neutron/python_tests/test_dvr.py [hunk 4]
```diff
@@ -253,3 +257,247 @@ def test_routing_after_reset_computes(self, prepare_openstack, env_name):
...
+                    result = remote.execute(cmd)
+                    if 'snat' in result['stdout']:
+                        snat_controller = node
+                        break
+        return snat_controller
+
+    def test_reschedule_router_from_snat_controller(self,
+                                                    create_dvr_router,
+                                                    create_net_and_vm):
+        """[Neutron DVR] Manually reschedule router from snat controller
```
Comment (TimurNurlygayanov) [inline mos_tests/neutron/python_tests/test_dvr.py:464]: Manually  > let's remove manually from here :)

**Comment by TregubovYury** [comment]
After applying of comments from pull#96 need to re-base with fresh master. will create new pull request to keep the history clear.

**Pull request closed without merging**